src/style.css

```diff
@@ -0,0 +1,87 @@
+body {
+  margin: 0;
+  font-family: Arial, sans-serif;
+  font-size: 100px;
+  color: white;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 300px;
+  flex-basis: 33%;
+  flex-grow: 1;
+}
+
+.block-1 {
+  background-color: rgb(255,0,0);
+  order: 1;
+}
+
+.block-2 {
+  background-color: rgb(204,0,0);
+  order: 2;
+}
+
+.block-3 {
+  background-color: rgb(153,0,0);
+  order: 3;
+}
+
+.block-4 {
+  background-color: rgb(102,0,0);
+  order: 6;
+}
+
+.block-5 {
+  background-color: rgb(51,0,0);
+  order: 5;
+}
+
+.block-6 {
+  background-color: rgb(0,0,0);
+  order: 4;
+}
+
+@media (max-width: 899px) {
+  .block {
+    flex-basis: 50%;
+  }
+
+  .block-3 {
+    order: 4;
+  }
+
+  .block-4 {
+    order: 3;
+  }
+
+  .block-6 {
+    background-color: black;
```

Comment:
This block doesn't change color depending on screen width.
So, the background-color rule here is redundant.

Reply:
Yes I think this i forgot to delete after copy pasting from code above default .block-6. I will delete it.
thank you